app/src/pages/admin/allUsersPage/allUsersGrid/nameColumn/nameColumn.jsx

```diff
@@ -46,7 +47,7 @@ const messages = defineMessages({
 @connect(
   (state) => ({
     currentUser: userIdSelector(state),
-    activeProject: activeProjectSelector(state),
+    projectKey: projectKeySelector(state),
```

Comment:
Do we really need to use this selector in admin page on users page?
TBD

Reply:
needed for UserAvatar

Reply:
Looks like the backend should update this endpoint to not use any project identifier at all.
![image](https://github.com/reportportal/service-ui/assets/31861805/2e7d77b7-3f31-435f-b93c-087bff83b855)
The reason is that the user can exists without any project and we still need to be able to fetch his photo.
Please, discuss with BA and create a task for WS+UI to handle this separately.